### ISSUE TITLE
S4651 Possible False Positive On "to left"

### ISSUE THREAD

**Issue by srvrguy** [SonarSource/sonar-css#201]
In a project that I'm analyzing, I seem to be tripping S4651, but as far as I and several other developers can tell, it seems to be a false positive because of the direction "to-left" not being considered valid. The particular CSS rule flagged is:

```background: -webkit-linear-gradient(to left, #ff0000 0%, #ffff00 16%, #00ff00 33%, #00ffff 50%, #0000ff 67%, #ff00ff 84%, #ff0004 100%);```

It's possible that the problem is because it's a vendor-prefixed rule, as the non-prefixed version seems to be fine. However, if that is the case then the description is misleading.

**Comment by vilchik-elena** [comment]
Hi @srvrguy,

In fact looks like `to` is not allowed in `-webkit` prefixed function (see [here](https://developer.mozilla.org/de/docs/Web/CSS/linear-gradient) in section "Cross-browser gradients").

I will add example with `-webkit` to rule description

**Comment by andrea-guarino-sonarsource** [comment]
Closing this issue because rule description has been updated.
